src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,242 @@
+package mate.jdbc.dao;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (model, color, price) "
+                + "VALUES (?, ?, ?) ";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement saveCarStatement = connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            saveCarStatement.setString(1, car.getModel());
+            saveCarStatement.setString(2, car.getColor());
+            saveCarStatement.setBigDecimal(3, car.getPrice());
+            saveCarStatement.executeUpdate();
+            ResultSet resultSet = saveCarStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+            return car;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't create "
+                    + car + ". ", throwable);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT \n"
+                + "    id as car_id, model, color, price\n"
+                + "FROM\n"
```

Comment:
```suggestion
                + "FROM "
```
для читабельності можна замість переносів просто пробіли ставити, переноси на синтаксис ніяк не впливають